protostar/protostar_cli.py

```diff
@@ -15,6 +17,13 @@
 from protostar.utils.log_color_provider import LogColorProvider
 
 
+def _consume_pythonpath():
```

Comment:
hmm, this function is not clearing nor removing the env var, so the term `consume` is misleading here. maybe:

```suggestion
def _apply_pythonpath():
```

Reply:
done